lib/ansible_tower_client/base_models/project.rb

```diff
@@ -19,8 +19,19 @@ def update
     end
 
     def last_update
-      return if related['last_update'].blank?
-      api.project_updates.find(related['last_update'])
+      return @last_update if defined? @last_update
+      return @last_update = nil unless related.raw_hash.key?('last_update')
+      return @last_update = nil if (update_id = related.last_update).blank?
+
+      unless update_id.kind_of?(Numeric) || update_id =~ /\A\d+\z/
```

Comment:
Minor, but I find the `unless ... || ...` construct confusing; this might be more readable as `if !update_id.kind_of?(Numeric) && update_id !~ /\A\d+\z/`

(Even that way, I find this conditional hard to read.)

Reply:
Two _nots_ in _if_, I guess rubocop will not be pleased? If it's hard for read anyway, I have an idea how to make you happier (I hope :smile:).

Reply:
done, commit https://github.com/ansible/ansible_tower_client_ruby/pull/123/commits/bd12dabe00f8eb4b4e0c4be680b4202778c9ffe0

Reply:
please tell me, who is boss here, you or rubocop? :smiley:  https://github.com/ansible/ansible_tower_client_ruby/pull/123#issuecomment-494736702